deployment/tasks/after_update_code.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install PHP dependencies
+  composer:
+    command: install
+    working_dir: "{{ project_dir }}"
+    no_dev: "{{ (stage == 'production') | ternary('Yes','No') }}"
+    optimize_autoloader: "yes"
+
+- name: Install JavaScript dependencies
+  npm:
+    path: "{{ project_dir }}"
+
+- name: Compile JavaScript
+  command: npm run build-js
+  args:
+    chdir: "{{ project_dir }}"
+
+- name: Remove node modules
```

Comment:
This became more complicated since as each skin can have its own node_modules.
Rational was to minimize the size of the dist file, could be done likewise of in a more clever way (project-specific ignore file fed to the role [not supported yet!] maybe?).